src/components/Form.js

```diff
@@ -27,25 +27,26 @@ const Form = () => {
   };
   return (
     <form className="form" id="book-form" onSubmit={handleSubmit}>
-      <h2 className="form-title">Add New Book</h2>
+      <div className="Line" />
+      <h2 className="form-title">ADD NEW BOOK</h2>
       <div className="form-rapper">
         <input
           type="text"
           className="add-book"
           name="title"
-          placeholder="Book title"
+          placeholder="BOOK TITLE"
```

Comment:
- [ ] Also your form is not justified to it's original place. Kindly make it's width 100% placed as the original :)

Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/230476430-2b1a29ee-49a1-4ecf-a8c7-afe0ee5e2d7a.png"> | <img src="https://user-images.githubusercontent.com/69011963/230476494-e10d8fea-eaff-4368-abf7-dbb807e1de54.png" width="1440"/>

Reply:
Hello @Rustamxon7 
Thank you for your constructive comments. I will fix it.